tests/devices/test_default_qubit_jax.py

```diff
@@ -512,6 +513,42 @@ def circuit():
 
         circuit()  # Just don't crash.
 
+    def test_parametrized_evolution_state_vector(self, mocker):
+        """Test that when executing a ParametrizedEvolution with ``num_wires >= device.num_wires/2``
+        the `_evolve_state_vector_under_parametrized_evolution` method is used."""
+        dev = qml.device("default.qubit.jax", wires=1, shots=1000)
+        H = ParametrizedHamiltonian([1], [qml.PauliX(0)])
+        spy = mocker.spy(dev, "_evolve_state_vector_under_parametrized_evolution")
+
+        @jax.jit
+        @qml.qnode(dev, interface="jax", diff_method=None)
```

Comment:
why diff method none?

Reply:
It probably got copied over. Removing it 👍